consts.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const THIRTEEN = 13;
+const THIRTEEN = 12;
```

Comment:
This is not 13

Reply:
It is in base 11.

Reply:
Really strong point. I'm closing this in favor of a more inclusive pr: https://github.com/jezen/is-thirteen/pull/474